### DISCUSSION TITLE
Add tabs from Radiance

### DISCUSSION THREAD

**Discussion by russellbanks** [kirill-grouchnikov/aurora#24] [Ideas]
Is it possible we could get tabs, similar to what is implemented in Radiance but native for JetBrains Compose?

<p align="center">
<img src="https://raw.githubusercontent.com/kirill-grouchnikov/radiance/sunshine/docs/images/theming/skins/nebula1.png" width="340" height="258" border=0>
<img src="https://raw.githubusercontent.com/kirill-grouchnikov/radiance/sunshine/docs/images/theming/skins/gemini2.png" width="340" height="258" border=0>
</p>

<p align="center">
<img src="https://raw.githubusercontent.com/kirill-grouchnikov/radiance/sunshine/docs/images/theming/skins/graphitechalk1.png" width="340" height="258" border=0>
<img src="https://raw.githubusercontent.com/kirill-grouchnikov/radiance/sunshine/docs/images/theming/skins/nightshade2.png" width="340" height="258" border=0>
</p>

**Comment by kirill-grouchnikov** [comment]
That would be a nice addition, following the overall look of Radiance with the functionality of https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/commonMain/kotlin/androidx/compose/material/TabRow.kt and https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/commonMain/kotlin/androidx/compose/material/Tab.kt

**Comment by kirill-grouchnikov** [comment]
Opened https://github.com/kirill-grouchnikov/aurora/issues/25
